src/views/Crowdfundings/index.tsx

```diff
@@ -93,7 +95,9 @@ function Proposals() {
       navigate(link);
     }
   };
-
+  function handleShowMore() {
+    setShowOnlyTen(showOnlyTen + 10);
+  }
   function calculateDaysAndHours(dateTimeStr: string) {
     const targetDateTime = new Date(dateTimeStr);
```

Comment:
I think here also needs the 'Z' added to the time since it shows up as the users timezone instead of UTC.